web/package.json

```diff
@@ -29,11 +29,11 @@
     "deploy": "DEPLOY_MODE=dev vue-cli-service build && node ./deploy-dev.js"
   },
   "dependencies": {
+    "imjoy-core": "^0.11.7",
```

Comment:
The rest of the items seem to be sorted 🔡. Can't we keep that order?

Reply:
I would like to have imjoy-core first to earn 0.5s time when bumping its version number ;)

Reply:
Done.